meta-psa/recipes-security/psa-trusted-storage-linux/psa-trusted-storage-linux_1.0.0.bb

```diff
@@ -32,3 +42,14 @@ RDEPENDS_${PN} = "\
     ecryptfs-utils \
     keyutils \
 "
+
```

Comment:
Can you please move the `RDEPENDS_${PN}` to the top of the file, next to the `DEPENDS`?
And `ecryptfs-utils` and `keyutils` are a dependency for the test package, not the main one. Right?